napari/layers/points/points.py

```diff
@@ -58,29 +58,29 @@ class Points(Layer):
         Coordinates for N points in D dimensions.
     symbol : str
         Symbol used for all point markers.
-    size : float
-        Size of the marker for the next point to be added or the currently
-        selected point.
+    size : array (N, D)
+        Array of sizes for each point in each dimension. Must have the same
```

Comment:
Thanks for fixing this! I just noticed this yesterday and hadn't gotten around to making an issue.